subaru_outback_2016_eyesight.dbc

```diff
@@ -34,19 +34,31 @@ NS_ :
 BS_:
 
 BU_: XXX
+VAL_TABLE_ OnOffList 1 "On" 0 "Off" ;
 
+BO_ 884 NEW_MSG_3: 8 XXX
+ SG_ DoorStatus : 31|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 324 NEW_MSG_3: 7 XXX
+ SG_ BrakeApplied : 8|1@0+ (1,0) [0|0] ""  XXX
+ SG_ CruiseSetButton : 3|1@0+ (1,0) [0|0] ""  XXX
+ SG_ CruiseResumeButton : 4|1@0+ (1,0) [0|0] ""  XXX
```

Comment:
can cruise Set and Resume buttons be controlled independently? if yes, then it's ok to keep them in separate signals. If not they should be merged in a 2 bits signal (in the Acura/Honda all the Cruise buttons are mutually exclusive).